### PR TITLE
appclone category

### PR DESCRIPTION
**Pull request by felipebaltazar**
## Submission description

- **Ui snippets**
Nubank app clone and a simple SkiaSharp chart.

- **New Category**
AppClone (I've updated some "snippets" with this new category.)

- **Preview**

<p align="center">
	<kbd>
		image
	</kbd>
</p>

<p align="center">
	<kbd>
		image
	</kbd>
</p>

### CODE REVIEW

File: src/Snppts/Snppts.csproj [hunk 1]
```diff
@@ -14,7 +14,6 @@
 
   <ItemGroup>
     <Folder Include="Views\Snippets\" />
-    <Folder Include="Snippets\" />
```
Comment (jfversluis) [inline src/Snppts/Snppts.csproj]: I've seen this on multiple PRs now, so it's probably something that happens automatically, but any idea why this is removed?

Comment (felipebaltazar) [reply src/Snppts/Snppts.csproj]: @jfversluis  humm.. that's strange...
Visually nothing was changed, but when I edit the "csproj" I can see this remotion

I've tried to download again and made some changes and nothing was removed 🤔
Btw, I've undone these changes

<p align="center">
	<kbd>
		image
	</kbd>
</p>

Comment (attrib75) [reply src/Snppts/Snppts.csproj]: > 
> 
> I've seen this on multiple PRs now, so it's probably something that happens automatically, but any idea why this is removed?

I've seen the same thing

**Review by jfversluis — APPROVED** [review]
Looking good! Thanks for your contribution!

**Comment by sthewissen** [comment]
Thanks for contributing to Snppts! Please fill in the following survey to receive your swag :)

https://www.surveymonkey.com/r/88ZQ6G2

**Pull request merged**